ci/checks/copyright.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
```

Comment:
The copyright file itself doesn't have a copyright...

🥲

Reply:
Lol, my bad